config/apinetlet/default/manager_auth_proxy_patch.yaml

```
@@ -24,3 +24,4 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
+        - "--api-net-namespace=apinetlet-system"
```

This is a potential problem, the net resources should be handled in `ironcore-net-system`.

```suggestion
        - "--api-net-namespace=ironcore-net-system"
```

---

This flag actually should not be part of the patch as the `namespace` is typically patched as n arg on the deployment size. Default behavior should be not setting this argument.